documentation/core/Modifiers.md

```diff
@@ -0,0 +1,20 @@
+# Modifiers
+A modifier is: ...
+
+The main modifiers are `Attribute` and `Format`.
+
+## Attributes
+When parsing and rendering a document, we need to save the attributes that a
+DOM node has and save it in our VNode.
+
+Attributes exists to:
+- retrieve HTML attributes that a DOM node has originally
+- render the HTML attributes that a VNode has
+
+## Format
+The DOM inline nodes (e.g. span, strong, em, b, a, ...) in HTML are difficult to
```

Comment:
The existence of a dichotomy between blocks and inline nodes is what causes a level of complexity that we managed to eliminate by introducing `Format`. It's not so much that the inline nodes themselves are hard to manipulate but the fact that there is that dichotomy means that we have to constantly be on our guard and treat different nodes differently, which can end up being complicated.